addusers.py

```diff
@@ -388,11 +389,22 @@ def parse_rows(rows):
                    ks_users.append(new_ks_user)
                    subscribe_emails.append(user.email)                
                 elif not ks_user:
-                    raise ItemNotFoundError('User', user.name)
+                    if user.is_requestor:
+                        raise ItemNotFoundError('User', user.name)
+                    else:
+                        # We don't treat this as a critical error
+                        print ("WARNING: Additional user {} does not exist. "
+                               "User will not be added to project {}."
+                               "Obtain the correct username and add the user"
+                               "manually.").format(user.name, 
+                                                   ks_project.name)
+                
                 else:
                     print "Adding existing user {} to project {}".format(ks_user.name, ks_project.name)
                     response = openstack.grant_role(auth_url, ks_project.id, ks_user.id, ks_member_role.id)
-                copy_index.append(user.row)
+                
+                if user.is_requestor: 
```

Comment:
This stops the script  from repeatedly marking the same row for deletion, which will cause neighboring rows to be deleted by accident.  This is the quick fix; I don't really like it.  But it works for now and hopefully a more elegant solution will come with the fix to #21